src/main/scala/com/horizon/exchangeapi/tables/Schema.scala

```diff
@@ -178,13 +178,19 @@ object SchemaTQ {
         sqlu"alter table servicepolicies add column label character varying not null default ''",
         sqlu"alter table servicepolicies add column description character varying not null default ''"
       )
+      case 41 => DBIO.seq(
+        sqlu"alter table patterns add column secretbinding character varying not null default ''"
+      )
+      case 42 => DBIO.seq(
```

Comment:
In the future, we try to keep it one DB upgrade step per PR. It's fine for now, but in the future these should be one step.